docs/animations.md

```diff
@@ -722,7 +722,7 @@ While using transform styles such as `rotateY`, `rotateX`, and others ensure the
 
 The RNTester app has various examples of `Animated` in use:
 
-- [AnimatedGratuitousApp](https://github.com/facebook/react-native/tree/master/packages/rn-tester/js/examples/Animated/AnimatedGratuitousApp)
+- [AnimatedGratuitousApp]()
```

Comment:
```suggestion
- [AnimatedGratuitousApp](https://github.com/facebook/react-native/tree/main/packages/rn-tester/js/examples/AnimatedGratuitousApp)
```
This is the correct link, besides the branch name change, it looks like the example has have been out from `Animated` folder.

Reply:
Thanks a lot, I should have read the link more carefully, it's so obvious now that the branch name changed to `main`. Thanks!